lisa/notifiers/log_file.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+from dataclasses import dataclass
+from typing import Any, List, Type, cast
+
+from dataclasses_json import dataclass_json
+
+from lisa import notifier, schema
+from lisa.environment import Environment
+from lisa.testsuite import TestResultMessage
+from lisa.util import constants, dump_file
+
+
+@dataclass_json()
+@dataclass
+class LogFileSchema(schema.TypedSchema):
+    log_level: str = logging.getLevelName(logging.DEBUG)
+
+
+class LogFile(notifier.Notifier):
+    """
+    It's a sample notifier, output subscribed message to a log file.
+    It can be used to troubleshooting what kind of message received.
+    """
+    
+
+    @classmethod
+    def type_name(cls) -> str:
+        return constants.NOTIFIER_LOG_FILE
+
+    @classmethod
+    def type_schema(cls) -> Type[schema.TypedSchema]:
+        return LogFileSchema
+
+    def _received_message(self, message: notifier.MessageBase) -> None:
```

Comment:
Refer to below logic, it should output the same format. Make the code shareable.

https://github.com/microsoft/lisa/blob/main/lisa/runner.py#L210